src/schemas/json/package.json

```diff
@@ -232,16 +232,12 @@
             ".": {
               "$ref": "#/definitions/packageExportsEntryOrFallback",
               "description": "The module path that is resolved when the module specifier matches \"name\", shadows the \"main\" field."
-            },
-            "./": {
-              "$ref": "#/definitions/packageExportsEntryOrFallback",
-              "description": "The module path prefix that is resolved when the module specifier starts with \"name/\", set to \"./\" to allow external modules to import any subpath."
             }
           },
           "patternProperties": {
-            "^\\./": {
+            "^\\./.+": {
               "$ref": "#/definitions/packageExportsEntryOrFallback",
-              "description": "The module path that is resolved when the path component of the module specifier matches the property name."
+              "description": "The module path prefix that is resolved when the module specifier starts with \"name/\", set to \"./*\" to allow external modules to import any subpath."
```

Comment:
I've reused the description from the property that I've removed - it holds more information. But I'm not sure if now this will be totally understandable for people. Let me know what do you think